app/views/posts/_post_details.html.erb

```diff
@@ -0,0 +1,13 @@
+<div class="post">
+  <div class="comment-header">
+    <h2>
+      Post by <%= post.author.name %>
+    </h2>
+    <small>
+      <span>Comment: <%= post.comments_counter %>, Likes: <%= post.likes_counter %></span>
```

Comment:
- [ ] I am not able to create comments please address that
![post](https://user-images.githubusercontent.com/44745855/172701658-8c7736fe-8194-475e-830a-a7d17fdd5200.png)